CHANGELOG.md

```diff
@@ -9,7 +9,9 @@ Breaking changes result in a different major. UI changes that might break custom
 ### Changed
 - Internal: Removed `preact-router`.
 - Public: Removed URL routes for each step of the SDK flow.
-- Internal: Removed unused components - Dropdown and ActionBar
+- Internal: Removed unused components - Dropdown and ActionBar.
+- Internal: Use the staging backend when in development.
+# TODO: release and bump onfido-sdk-core version.
```

Comment:
`#` creates a title on markdown, not a comment. comments are a bit noisy in markdown, see: http://stackoverflow.com/questions/4823468/comments-in-markdown